dashboard/config/courses/csd-pilot.course

```diff
@@ -16,7 +16,7 @@
         "https://forum.code.org/c/csd"
       ]
     ],
-    "version_year": "2020",
+    "version_year": "2020-pilot",
```

Comment:
Changed this because otherwise this conflicts with `csd-2020.course`. Not sure if this uniqueness constraint is a problem, or if it's ok to change this .course file.

I could also see deleting this file if it's not needed anymore, or changing the `family_name` to something else.

Reply:
the CSP pilot scripts should definitely not be in the CSD course family. it should be ok to remove them and not let them have a family name or version year.

Reply:
This is the CSD pilot, right? So that part makes sense.

Reply:
Added https://codedotorg.atlassian.net/browse/PLAT-276 to up next.